src/http/response.ts

```diff
@@ -19,11 +19,13 @@ export enum LkErrorKey {
   // Business and access errors
   FEATURE_DISABLED = 'feature_disabled',
   UNAUTHORIZED = 'unauthorized',
+  LDAP_BIND_ERROR = 'ldap_bind_error',
   DATA_SOURCE_UNAVAILABLE = 'dataSource_unavailable',
   GUEST_DISABLED = 'guest_disabled',
   FORBIDDEN = 'forbidden',
   NOT_FOUND = 'not_found',
   BAD_GRAPH_REQUEST = 'bad_graph_request',
+  CREATION_FAILED = 'creation_failed',
```

Comment:
Unless I'm wrong, this error key is not used, so there is no need to add it here.